docs/content/configuration.md

```diff
@@ -387,6 +387,13 @@ keys:
     private_key: ${BUNDLE_SERVICE_SIGNING_KEY}
 ```
 
+{{% danger %}}
+OPA masks services authentication secrets which make use of the `credentials` field, in order to prevent the exposure of sensitive tokens.
```

Comment:
Sorry there's something broken with the shortcodes, could you please do
```suggestion
OPA masks services authentication secrets which make use of the <code>credentials</code> field, in order to prevent the exposure of sensitive tokens.
```
?